runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationSections.module.scss

```diff
@@ -0,0 +1,160 @@
+@import "#src-landing/scss/main.scss";
+
+$ARROW_ICON_WIDTH: 300px;
+
+.root {
+  background-color: $palette-accent-primary;
+  position: relative;
+}
+
+.arrowIconWrapper {
+  width: $ARROW_ICON_WIDTH;
+  position: absolute;
+}
+
+.arrowIcon {
+  width: 100%;
+  height: 100%;
+  filter: brightness(0) invert(1);
+}
+
+.centeredWrapper {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding-top: $padding-10;
+  padding-bottom: $padding-10;
+  padding-left: $padding-3;
+  padding-right: $padding-3;
+}
+
+.title {
+  @include font-face-base;
+  color: $text-color-light;
+  font-weight: $font-weight-bold;
+  font-size: $font-size-h2;
+}
+
+.fontDark {
+  color: $text-color-dark;
+}
+
+.logosGrid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: $padding-4;
+  padding-top: $padding-3;
+  padding-bottom: $padding-3;
+}
+
+.logosGrid div {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: $padding-4;
+  width: 100%;
+  height: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  transition: all 250ms ease;
+  &:hover {
+    transform: scale(1.1);
+  }
+}
+
+.sapLogo {
+  background-image: url("/images/logos/sap-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/sap-logo.svg");
+  }
+}
+
+.beeofficeLogo {
+  background-image: url("/images/logos/beeoffice-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/beeoffice-logo.svg");
+  }
+}
+
+.sharepointLogo {
+  background-image: url("/images/logos/sharepoint-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/sharepoint-logo.svg");
+  }
+}
+
+.excelLogo {
+  background-image: url("/images/logos/excel-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/excel-logo.svg");
+  }
+}
+  
+.jiraLogo {
+  background-image: url("/images/logos/jira-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/jira-logo.svg");
+  }
+}
+
+.googlesheetsLogo {
+  background-image: url("/images/logos/googlesheets-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/googlesheets-logo.svg");
+  }
+}
+  
+.asanaLogo {
+  background-image: url("/images/logos/asana-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/asana-logo.svg");
+  }
+}
+  
+.powerpointLogo {
+  background-image: url("/images/logos/powerpoint-logo-light.svg");
+  &:hover {
+    background-image: url("/images/logos/powerpoint-logo.svg");
+  }
+}
+
+@media only screen and (min-width: $breakpoint-xs){
```

Comment:
I see you went with mobile-first approach to writing css, i went the other way around. Can we talk when you are available?

Reply:
done, as we "discussed" xD